examples/textarea/client.js

```diff
@@ -11,17 +11,23 @@ var statusSpan = document.getElementById('status-span');
 statusSpan.innerHTML = 'Not Connected';
 
 element.style.backgroundColor = 'gray';
+var onopen = socket.onopen;
 socket.onopen = function() {
```

Comment:
It would be cleaner to use the [WebSocket events](https://developer.mozilla.org/en-US/docs/Web/API/WebSocket#Events) here, since these aren't intending to override behavior, they're just looking to add additional UI when something happens.
```js
socket.addEventListener('open', ...);
```

Reply:
Ha sorry old IE habit